### ISSUE TITLE
When melting metal dust, '1 dust -> 1 ingot value molten metal' crafting remains.

### ISSUE THREAD

**Issue by HardFur** [hadron13/Back-On-Track#2]
i solved that problem adding this in recipes.js :

```javascript
    event.remove({ id: TC('smeltery/melting/metal/iron/dust') })
    event.remove({ id: TC('smeltery/melting/metal/copper/dust') })
    event.remove({ id: TC('smeltery/melting/metal/tin/dust') })
    event.remove({ id: TC('smeltery/melting/metal/zinc/dust') })
    event.remove({ id: TC('smeltery/melting/metal/nickel/dust') })
    event.remove({ id: TC('smeltery/melting/metal/lead/dust') }) 
    //etc...
```

**Comment by HardFur** [comment]
and also metal block casting needs too many molten metal.

**Comment by hadron13** [comment]
The dust problem was fixed in 1.1.3e, but i don't remember checking the block casting problem.
